enterprise/dev/ci/internal/ci/operations.go

```diff
@@ -815,3 +815,11 @@ func uploadBuildeventTrace() operations.Operation {
 		)
 	}
 }
+
+func prPreview() operations.Operation {
+	return func(pipeline *bk.Pipeline) {
+		pipeline.AddStep(":globe_with_meridians: Trigger render.com PR Preview",
```

Comment:
To keep it short similarly to other step names:

```suggestion
		pipeline.AddStep(":globe_with_meridians: Client PR preview",
```